scripts/generate_conf_files.sh

```diff
@@ -98,5 +109,5 @@ echo "Nameserver is: $NAMESERVER"
 
 echo "Copying nginx config"
 envsubst '${ROOT_CA_CERT} ${ROOT_CA_KEY} ${LOG_DIR} ${PROXY_BUFFER_SIZE} ${PROXY_BUFFERS} ${PROXY_BUSY_BUFFERS_SIZE}' < ./conf/nginx.conf.template > ./conf/nginx.conf
-envsubst '${NAMESERVER} ${LOG_DIR} ${CONTENT_CACHE_DIR} ${CONTENT_CACHE_KEYS_ZONE} ${CONTENT_CACHE_SIZE} ${SSL_VERIFY_DEPTH}' < ./conf/nginx.conf.server.template > ./conf/nginx.conf.server
+envsubst '${PROXY_CONNECT_DATA_TIMEOUT} ${PROXY_READ_DATA_TIMEOUT} ${NAMESERVER} ${LOG_DIR} ${CONTENT_CACHE_DIR} ${CONTENT_CACHE_KEYS_ZONE} ${CONTENT_CACHE_SIZE} ${SSL_VERIFY_DEPTH}' < ./conf/nginx.conf.server.template > ./conf/nginx.conf.server
```

Comment:
@jschr I expected that these variables should be used during runtime but it seems that they are used during image build time. Tried to set:
<img width="1707" alt="Screenshot 2024-04-22 at 16 45 03" src="https://github.com/mirainc/funes/assets/3447630/8e5367bb-dba8-4fcf-8023-a320891ee03e">

after reboot nginx configuration was the same as default (60s):
<img width="930" alt="Screenshot 2024-04-22 at 16 44 21" src="https://github.com/mirainc/funes/assets/3447630/aa4a8182-496a-41fd-b8df-51b1afcf277b">

Did I do something wrong or this is expected behavior?

Reply:
It looks like we should be calling `run.sh` on the device here when starting `funes`: https://github.com/mirainc/screenray/blob/staging/funes/main.go#L93

Then `run.sh` calls `generate_conf_files.sh`, which _should_ pickup the environment variables at runtime. 

Can you confirm if the device variables are set as you would expect in Balena?